client/my-sites/posts/post-controls.jsx

```diff
@@ -1,226 +1,242 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	PureRenderMixin = require( 'react-pure-render/mixin' ),
-	url = require( 'url' );
+import React, { PropTypes } from 'react';
+import PureComponent from 'react-pure-render/component';
```

Comment:
As of [React 15.3.0](https://github.com/facebook/react/blob/master/CHANGELOG.md#1530-july-29-2016), you can now `import { PureComponent } from 'react';` . We'll plan to convert existing `react-pure-render` to the available equivalents and remove it as a dependency altogether.